patches/v5/4-slices-api.patch

```diff
@@ -29,21 +29,38 @@ index db2b6a78c1..f0e7745e01 100644
 +  [key: string]: SerializableProps
 +}
 +
-+export interface SliceProps {
++/**
++ * A props object for [slice placholder](https://v5.gatsbyjs.com/docs/reference/built-in-components/gatsby-slice/)
++ */
++export interface SlicePlaceholderProps {
 +  alias: string
 +  allowEmpty?: boolean
++  children?: React.ReactNode
 +  [key: string]: SerializableProps
 +}
 +
 +/**
-+ * TODO
```

Comment:
Should the TODO be removed? Otherwise can we add a comment what the TODO is for

Reply:
@tyhopp the TODO is removed, the diff is just weird :)

Reply:
Ahh the double diff got me!